src/Uno.UI.Tests/Windows_UI_XAML_Controls/GridTests/Given_Grid.cs

```diff
@@ -349,43 +457,58 @@ public void When_Grid_Has_Two_Colums_And_Two_Rows_And_No_Size_Spec()
 			SUT.RowDefinitions.Add(new RowDefinition { Height = "*" });
 
 			var c1 = SUT.AddChild(
-				new View { Name = "Child01", RequestedDesiredSize = new Windows.Foundation.Size(10, 10) }
+				new View
+					{
+						Name = "Child01",
+						RequestedDesiredSize = new Size(10, 10)
+					}
 				.GridPosition(0, 0)
 			);
 
 			var c2 = SUT.AddChild(
-				new View { Name = "Child02", RequestedDesiredSize = new Windows.Foundation.Size(10, 10) }
+				new View
+					{
+						Name = "Child02",
+						RequestedDesiredSize = new Size(10, 10)
+					}
 				.GridPosition(0, 1)
 			);
 
 			var c3 = SUT.AddChild(
-				new View { Name = "Child03", RequestedDesiredSize = new Windows.Foundation.Size(10, 10) }
+				new View
+					{
+						Name = "Child03",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'Child03' 5 times.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826507)